base/src/schema.test.ts

```diff
@@ -967,3 +1186,59 @@ test('findRelation', () => {
     ['tours', 'items', 'relation'],
   ])
 })
+
+describe('getSchemaDefinition', () => {
```

Comment:
Per the comment I gave earlier, you'll need to move this test group:

They should be under validating an object.